packages/core/src/_resources.scss

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+// Licensed under the BSD-3 License as modified (the “License”); you may obtain a copy
+// of the license at https://github.com/palantir/blueprint/blob/master/LICENSE
+// and https://github.com/palantir/blueprint/blob/master/PATENTS
+
+/*
+Resources
+
+Weight: 10
+
+Styleguide resources
+*/
+
+/*
+Sketch assets
```

Comment:
is there anything to say about these files? some intro to using them / what to expect?